core/standard/clause_7_building_blocks.adoc

```diff
@@ -492,10 +492,34 @@ include::requirements/record-core-query-parameters/REQ_query-param-limit.adoc[]
 [[core-query-parameters-q]]
 ===== Parameter q
 
-include::requirements/record-core-query-parameters/REQ_query-param-q-definition.adoc[]
+The `q` parameter is meant to provide and rudimentary, easy to implement keyword search capability across one or more text fields in a record.  The specific text fields in a record that are to be searched is not mandated in this standard.
 
 include::recommendations/record-core-query-parameters/REC_param-q.adoc[]
 
+The `q` parameter is a comma-separated list of search terms.  If any of these terms appear in a searched text field of a record then that record can be included in the result set (logical OR).  If any search term specified using the `q` parameter contains one or more spaces, this implied that all of the specified search terms must appear in a searched text field of a record before that record can be included in the result set (logical AND).
```

Comment:
Another, more natural interpretation of spaces in a search term would be that the search term including the spaces would match. That is, in the example `anytext CONTAINS 'climate change'`.

Both options are useful, but I would use the more intuitive option and leave the logical expressions to CQL2. That said, if there is a preference for the AND interpretation, that is OK for me, too.

Reply:
@pvretano Not that I can "see" the four spaces in the markdown rendering 😉 , but I agree that it should be any whitespace separating the words.

Reply:
I agree. I'd expect that if I search for `q=climate change` that I'd get all results that contain "climate change", but not the results that contain "change climate"